src/panels/calendar/ha-big-calendar.js

```diff
@@ -0,0 +1,65 @@
+import { html } from '@polymer/polymer/lib/utils/html-tag.js';
+import { PolymerElement } from '@polymer/polymer/polymer-element.js';
+
+/* polymer-lint disable */
+import { render } from 'react-dom';
+import React from 'react';
+/* polymer-lint enable */
+import BigCalendar from 'react-big-calendar';
+import moment from 'moment';
+
+import '../../resources/ha-style.js';
+
+BigCalendar.setLocalizer(BigCalendar.momentLocalizer(moment));
+
+class HABigCalendar extends PolymerElement {
+  static get template() {
+    return html`
+      <link rel="stylesheet" href="/static/panels/calendar/react-big-calendar.css">
+      <style include="iron-flex ha-style">
```

Comment:
Any reason you're including `iron-flex` and `ha-style`? They don't seem to be used.